<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>classeditor_form</class>
 <widget class="QDialog" name="classeditor_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Class Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Object class definition</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="classlevel1_comboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="classlevel2_comboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="classlevel3_comboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="classlevel4_comboBox"/>
      </item>
     </layout>
     <zorder>classlevel4_comboBox</zorder>
     <zorder>classlevel3_comboBox</zorder>
     <zorder>classlevel1_comboBox</zorder>
     <zorder>classlevel2_comboBox</zorder>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="classlevel4_label_2">
        <property name="text">
         <string>Editor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editor_lineEdit"/>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>You can also activate this form with F11 function key</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>buttonBox</zorder>
     <zorder>classlevel4_label_2</zorder>
     <zorder>editor_lineEdit</zorder>
     <zorder>label_3</zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Table column definition</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="classlevel1_label">
          <property name="text">
           <string>Class Level 1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="classlevel1_fields_comboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="classlevel2_label">
          <property name="text">
           <string>Class Level 2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="classlevel2_fields_comboBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="classlevel3_label">
          <property name="text">
           <string>Class Level 3</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="classlevel3_fields_comboBox"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="classlevel4_label">
          <property name="text">
           <string>Class Level 4</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="classlevel4_fields_comboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Console</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>For all selected elements set the value of field:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>0</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>classeditor_form</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>classeditor_form</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
